portfolio/src/main/webapp/style.css

```
@@ -4,6 +4,60 @@
   width: 650px;
 }
 
-#greeting-container {
-  margin-top: 20px;
+html{
```

Include a space between the selector and the declaration block. Here and throughout the file.

```suggestion
html {
```

https://google.github.io/styleguide/htmlcssguide.html#Declaration_Block_Separation

---

Ack!